package.json

```diff
@@ -1,84 +1,90 @@
 {
-    "name": "html-flip-book",
-    "version": "1.0.0",
-    "description": "Flipbook component for HTML",
-    "homepage": "https://github.com/doradsoft/html-flip-book",
-    "repository": {
-      "type": "git",
-      "url": "git+https://github.com/doradsoft/html-flip-book.git"
+  "name": "html-flip-book",
+  "version": "1.0.0",
+  "description": "Flipbook component for HTML",
+  "homepage": "https://github.com/doradsoft/html-flip-book",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/doradsoft/html-flip-book.git"
+  },
+  "bugs": {
+    "url": "https://github.com/doradsoft/html-flip-book/issues"
+  },
+  "engines": {
+    "node": ">=20"
+  },
+  "scripts": {
+    "bundle": "npm run format:write && npm run package",
+    "ci-test": "npx jest",
+    "coverage": "npx make-coverage-badge --output-path ./badges/coverage.svg",
+    "format:write": "npx prettier --write .",
+    "format:check": "npx prettier --check .",
+    "lint": "npx eslint . -c ./eslint.config.mjs",
+    "package": "npx ncc build src/index.ts -o dist --source-map --license licenses.txt",
+    "package:watch": "npm run package -- --watch",
+    "test": "npx jest",
+    "all": "npm run format:write && npm run lint && npm run test && npm run coverage && npm run package"
+  },
+  "compilerOptions": {
+    "moduleResolution": "nodenext",
+    "module": "nodenext"
+  },
+  "author": "DoradSoft",
+  "license": "MIT",
+  "jest": {
```

Comment:
please refactor to a separate config file.
https://jestjs.io/docs/configuration#:~:text=It%20is%20recommended%20to%20define%20the%20configuration%20in%20a%20dedicated%20JavaScript